packages/lodestar-params/src/beaconParams.ts

```diff
@@ -0,0 +1,86 @@
+import {IBeaconParams} from "./interface";
+import {
+  ContainerType, BigIntUintType, NumberUintType, BitVectorType, ByteVectorType,
+} from "@chainsafe/ssz";
+
+const Number64 = new NumberUintType({byteLength: 8});
+const BitVector8 = new BitVectorType({length: 8});
+const ByteVector32 = new ByteVectorType({length: 4});
+const ByteVector256 = new ByteVectorType({length: 32});
+const BigInt64 = new BigIntUintType({byteLength: 8});
+const BigInt256 = new BigIntUintType({byteLength: 32});
+
+export const BeaconParams = new ContainerType<IBeaconParams>({
+  fields: {
+    // Misc
+    MAX_COMMITTEES_PER_SLOT: Number64,
+    TARGET_COMMITTEE_SIZE: Number64,
+    MAX_VALIDATORS_PER_COMMITTEE: Number64,
+    MIN_PER_EPOCH_CHURN_LIMIT: Number64,
+    CHURN_LIMIT_QUOTIENT: Number64,
+    SHUFFLE_ROUND_COUNT: Number64,
+    MIN_GENESIS_TIME: Number64,
+    HYSTERESIS_QUOTIENT: Number64,
+    HYSTERESIS_DOWNWARD_MULTIPLIER: Number64,
+    HYSTERESIS_UPWARD_MULTIPLIER: Number64,
+    MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: Number64,
+    TARGET_AGGREGATORS_PER_COMMITTEE: Number64,
+    RANDOM_SUBNETS_PER_VALIDATOR: Number64,
+    EPOCHS_PER_RANDOM_SUBNET_SUBSCRIPTION: Number64,
+    SECONDS_PER_ETH1_BLOCK: Number64,
+
+    // Deposit contract
+    DEPOSIT_CONTRACT_ADDRESS: ByteVector256,
```

Comment:
should be bytevector32

Reply:
changed to ByteVector20 as per latest spec config